### DISCUSSION TITLE
Sostiuire Tuya con zigbee

### DISCUSSION THREAD

**Discussion by cecov74** [indomus/forum#2251] [HUB: Home Assistant]
Ciao a tutti, ho 3 prese smart che usano tuya, mi sono registrato al cloud ma dopo 3 mesi è scaduto, ho rimediato le key per bypassare ma mi mancano diverse informazioni, praticamente riesco solo ad accendere e spegnere, quindi ho pensato di acquistare un zigbee ma ne anche l'ultimo (sonoff  dungle usb zigbee) non trova i dispositivi, quale posso prendere per risolvere? Vorrei poter gestire le prese (accensione , watt,ecc) senza passare per il cloud.
Grazie

**Comment by 50494554524F** [comment]
è cambiato qualcosa?!? prima l'account tuya si poteva rinnovare ogni 12 mesi sempre gratuitamente...
